addons/udes_stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -24,6 +24,26 @@ class StockPickingBatch(models.Model):
             "in_progress": [("readonly", False)],
         },
     )
+    u_user_ids = fields.Many2many(
```

Comment:
Should we consider having only one field u_user_ids now, and removing Responsible User user_id field, or is a big change?

Reply:
Very good point, but yes it is a change too big, it will go its own story.